api/src/organizational-entities/infrastructure/repositories/organization-for-admin.repository.js

```diff
@@ -216,15 +216,10 @@ const update = async function (organization) {
     .where({ id: organization.id });
 };
 
-/**
- * @typedef {Object} OrganizationForAdminRepository
- * @property {archive} archive
- * @property {exist} exist
- * @property {findChildrenByParentOrganizationId} findChildrenByParentOrganizationId
- * @property {get} get
- * @property {save} save
- * @property {update} update
- */
+const deleteById = async function (id) {
+  const deleteCount = await knex(ORGANIZATIONS_TABLE_NAME).where({ id }).del();
+  return deleteCount === 1;
+};
```

Comment:
Suggestion de nommage d'argument `organizationId` plus explicite et qui est plus la norme dans la base de code.

De plus le `deleteCount` ne sert jamais à rien dans le code de production (c'est à dire hors tests), en effet soit la suppression est un succès, soit il y a une erreur et une exception sera levée. Mettre à jour les tests pour ne pas chercher à utiliser la valeur de retour booléenne précédente. Enfin, les tests testent déjà bien que la suppression est effective.

```suggestion
const deleteById = async function (organizationId) {
  await knex(ORGANIZATIONS_TABLE_NAME).where({ id: organizationId }).del();
};
```

Reply:
Bien vu, je devais le changer mais cela n'avait pas été fait!